modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/GroupResourceManagerTest.java

```diff
@@ -432,8 +485,8 @@ public void testDeleteGroupFails(String id, int expectedScimResponseStatus)
         if (expectedScimResponseStatus == ResponseCodeConstants.CODE_INTERNAL_ERROR) {
 
             abstractResourceManager
-            .when(() ->
-                    AbstractResourceManager.encodeSCIMException(any(InternalErrorException.class)))
+                    .when(() ->
```

Comment:
merge this into 1 line

Reply:
Fixed in commit 34423db0111992d6450b72b5cb1f2b50ab84aec0